src/Servers/HttpSys/src/RequestQueueMode.cs

```diff
@@ -22,6 +22,10 @@ public enum RequestQueueMode
         /// Create a queue with the given name if it does not already exist, otherwise attach to the existing queue.
         /// Most configuration options do not apply when attaching to an existing queue.
         /// </summary>
-        CreateOrAttach
+        CreateOrAttach,
+        /// <summary>
+        /// Attach to an existing queue to delegate transfer requests to.
+        /// </summary>
+        Receiver
```

Comment:
Oh, this shouldn't be public should it? It's only used in an internal API. Let's separate this to a new parameter on the request queue.